### ISSUE TITLE
fossa build appears to ignore --config -c (strace)

### ISSUE THREAD

**Issue by ava-dylang** [fossas/fossa-cli#150]
```$ strace -fetrace=file fossa -c .config/fossa.yml build  2>&1  | grep fossa
execve("/bin/fossa", ["fossa", "-c", ".config/fossa.yml", "build"], [/* 57 vars */]) = 0
[pid 30575] newfstatat(AT_FDCWD, "fossa", 0xc42013b3b8, 0) = -1 ENOENT (No such file or directory)
[pid 30575] newfstatat(AT_FDCWD, ".fossa.yml", 0xc42013b968, 0) = -1 ENOENT (No such file or directory)
[pid 30575] newfstatat(AT_FDCWD, ".fossa.yaml", 0xc42013ba38, 0) = -1 ENOENT (No such file or directory)
```

**Comment by ilikebits** [comment]
Yes, this is a bug in the current (too complex) configuration logic. It should be resolved in the v0.7.0 branch.

**Comment by ilikebits** [comment]
This is now fixed in `v0.7.0-beta.1`.
